docs/configuration-reference/platforms/packet.md

```diff
@@ -205,7 +205,7 @@ node_type = var.custom_default_worker_type
 | `management_cidrs`                    | List of IPv4 CIDRs authorized to access or manage the cluster. Example ["0.0.0.0/0"] to allow all.                                                                            | -               | true     |
 | `node_private_cidr`                   | Private IPv4 CIDR of the nodes used to allow inter-node traffic. Example "10.0.0.0/8"                                                                                         | -               | true     |
 | `enable_aggregation`                  | Enable the Kubernetes Aggregation Layer.                                                                                                                                      | true            | false    |
-| `networking`                          | CNI network plugin ("flannel" or "calico").                                                                                                                                   | "calico"        | false    |
+| `networking`                          | CNI network plugin (only "calico" supported on Packet).                                                                                                                       | "calico"        | false    |
```

Comment:
Why can't we remove `networking` parameter completely?

Reply:
I was thinking on future networking that we might want to add so we don't have to re-add this later, but I'm not sure if that's a good enough reason...

Reply:
I'm a bit concerned, that we still require user configuration block for it, even though it's the only valid option. We could leave it and make it a default, so user don't have to define it maybe?

Reply:
This is already the case, if you don't specify it in lokocfg it will take the terraform default which is calico.

Reply:
Ah yes, I misread the code. I think it's fine then to leave it.